tests/mocks/mock-stacks-txs.ts

```diff
@@ -0,0 +1,33 @@
+import type { Page } from '@playwright/test';
+
+import { TEST_ACCOUNT_1_STX_ADDRESS } from './constants';
+
+export async function mockMainnetTestAccountStacksTxsRequests(page: Page) {
+  await Promise.all([
+    page.route(
+      `**/api.hiro.so/extended/v1/address/${TEST_ACCOUNT_1_STX_ADDRESS}/transactions_with_transfers?limit=50`,
```

Comment:
Maybe we can use a constant / types for the URL?

Reply:
yeah, thanks, I'll add in next pr